packages/components/src/components/modal/__snapshots__/modal-styles.spec.ts.snap

```diff
@@ -32,7 +32,7 @@ exports[`getComponentCss() should return correct css for open: false, fullscreen
   opacity: 0 !important;
   backdrop-filter: blur(0px) !important;
   -webkit-backdrop-filter: blur(0px) !important;
-  transition: opacity .2s cubic-bezier(.16,1,.3,1), backdrop-filter .2s cubic-bezier(.16,1,.3,1), --webkit-backdrop-filter .2s cubic-bezier(.16,1,.3,1) !important;
+  transition: opacity var(--p-transition-duration, 0.25s) cubic-bezier(0.25,0.1,0.25,1), backdrop-filter var(--p-transition-duration, 0.25s) cubic-bezier(0.25,0.1,0.25,1), -webkit-backdrop-filter var(--p-transition-duration, 0.25s) cubic-bezier(0.25,0.1,0.25,1) !important;
```

Comment:
Missing leading `-` in `-webkit-backdrop-filter`?
This was `--webkit-backdrop-filter` before.

Reply:
as discussed in private chat `-webkit-backdrop-filter` is correct https://caniuse.com/?search=-webkit-backdrop-filter